docs/DEVELOPING.md

```diff
@@ -276,3 +276,7 @@ To implement a new source, check [CONTRIBUTE.rst](./CONTRIBUTING.md)
 
 - MacOS or Linux server. The connector has been tested with CentOS 7, MacOS Monterey v12.0.1.
 - Python version 3.10 or later.
+- To fix SSL certificate verification failed error, users have to run this to connect with Amazon S3:
```

Comment:
I would add a section about installing Python 3.10, and for macOS this extra step, independently from the S3 connector.

Reply:
So if you are planning to add a section regarding the installation, do you want me to close this PR ?

Reply:
no that's fine as a first step, thanks!